invenio_query_parser/utils.py

```diff
@@ -68,17 +68,33 @@ def generate_keywords():
         from invenio_base.globals import cfg
```

Comment:
Can you rename the function to: something more meaningful such as `generate_valid_keywords`, and add a docstring? (I should have commented on this in previous commit, sorry! :smile: )